### PR TITLE
Add job details for eda-server usage

### PR DESCRIPTION
**Pull request by hsong-rh**
Add more job related details to the event queue, so EDA server side can save them for either debug or job rerun.

Resolves: [AAP-5081](https://issues.redhat.com/browse/AAP-5081)

### CODE REVIEW

File: ansible_rulebook/builtin.py [hunk 1]
```diff
@@ -274,19 +303,33 @@ async def run_module(
...
     )
+    job_id = str(uuid.uuid4())
+
+    await event_log.put(
+        dict(
+            type="Job",
+            job_id=job_id,
+            ansible_events_id=settings.identifier,
+            name=module_name,
+            ruleset=ruleset,
```
Comment (mkanoor) [inline ansible_rulebook/builtin.py:326]: @hsong-rh This should be the source_ruleset_name

File: tests/test_rules.py [hunk 2]
```diff
@@ -180,9 +180,9 @@ async def test_generate_rules():
     print(durable_rulesets[0][0].define())
     assert_fact("Demo rules", {"payload": {"text": "hello"}})
 
-    assert ruleset_queue_plans[0][2].get_nowait()[1] == "slack"
-    assert ruleset_queue_plans[0][2].get_nowait()[1] == "assert_fact"
-    assert ruleset_queue_plans[0][2].get_nowait()[1] == "log"
+    assert ruleset_queue_plans[0][2].get_nowait().action == "slack"
```
Comment (mkanoor) [inline tests/test_rules.py:183]: @hsong-rh Can one of these tests be modified to check the ruleset name and the rule name?

**Review by mkanoor — CHANGES_REQUESTED** [review]
Use the correct ruleset the current implementation breaks when using multiple rulesets

File: ansible_rulebook/builtin.py [hunk 3]
```diff
@@ -260,6 +287,8 @@ async def run_module(
     variables: Dict,
     facts: Dict,
     ruleset: str,
+    source_ruleset_name: str,
```
Comment (mkanoor) [inline ansible_rulebook/builtin.py:290]: @hsong-rh This seems to be off by one
The ruleset should be after source_ruleset_name and source_rule_name

**Review by mkanoor — APPROVED** [review]

**Comment by hsong-rh** [comment]
@benthomasson can you review and merge this? thanks

**Review by cutwater — APPROVED** [review]

**Pull request merged**